operator/src/main/resources/scripts/startServer.sh

```diff
@@ -34,15 +34,17 @@ function copyIfChanged() {
     trace "Copying '$1' to '$2'."
     cp $1 $2
     [ $? -ne 0 ] && trace SEVERE "failed cp $1 $2" && exitOrLoop
-    chmod 750 $2 
-    [ $? -ne 0 ] && trace SEVERE "failed chmod 750 $2" && exitOrLoop
+    if [ -O "$2" ]; then
+      chmod 770 $2
+      [ $? -ne 0 ] && trace SEVERE "failed chmod 770 $2" && exitOrLoop
+    fi
```

Comment:
What file isn't owned?  Note that there is similar logic in the liveness probe script.

Reply:
On OpenShift, the containers will not be run as the "oracle" user, but instead will be run as a new user (large UID), but that is part of the "root" group. In the case of the boot.properties, or any file that already exists and is being copied over, the file will still be owned by "oracle" and so the script cannot chmod the file.

Reply:
Yes, I'll change the liveness probe too

Reply:
It's interesting that boot.properties is created and copied over by the operator at runtime (contains the WL credentials), but it still needs an unknown check and a chmod...

Reply:
It's not copied by the "operator". It's copied by the user principal inside the server instance running a script injected by the operator.